src/test/resources/correct_path.csv

```diff
@@ -0,0 +1,2 @@
+ID,NAME
+1,Bob
```

Comment:
Empty line

Reply:
let's add testing for file with with the same headers as in amazon reviews. For example you may copy first 10 lines from there